src/App.scss

```diff
@@ -10,7 +10,7 @@ body {
   width: 100vw;
   background-color: #ffffff;
   color: $fontColor;
-  font-family: $hackUCIFont;
+  font-family: $fontFamilyBase;
```

Comment:
With proper usage of Bootstrap's type system, we can avoid the need to specify the main font family.

Reply:
Updated.